src/schema/sources.ts

```diff
@@ -824,7 +850,7 @@ const updateHideFeedPostsFlag = async (
     { sourceId, userId: ctx.userId },
     {
       flags: updateFlagsStatement<SourceMember>({
-        hideFeedPosts: value,
+        [flag]: value,
       }),
     },
   );
```

Comment:
Even though code is repeated I don't see much use for this helper anymore. For example if you need to update more flags it would be of no use since we would not want to do multiple queries. 

The original helper was added by me per review feedback of duplicate code but I would just do it inline at this point.

Reply:
I agree, what are your thoughts on having one mutation to update all flags to reduce repetition here?

Reply:
We might need different logic validation for different flags so updating them all in one mutation would mean we would also have a lot of if-else depending on which flags are updated.

Reply:
I think all calls can only update one at the time anyway right?
Just noting as a refactor might be too much work at this point as this would work?

Reply:
good points, I also think a more descriptive API is better, even though it can become more verbose. A mutation called `collapsePinnedPosts` is self-documenting vs. something like `editSourceMemberFlags`

Reply:
I would say leave `updateHideFeedPostsFlag` as it is and do the whatever you need directly in your mutations to not change parts of the code not connected to the PR and minimize changes, what do you guys think?

Reply:
updated 🙏